*** catalina.bat_default	2015-11-17 14:38:37.404307993 +0300
--- catalina.bat_modified	2015-11-17 14:38:49.160307983 +0300
***************
*** 193,204 ****
  if not exist "%CATALINA_BASE%\conf\logging.properties" goto noJuliConfig
  set LOGGING_CONFIG=-Djava.util.logging.config.file="%CATALINA_BASE%\conf\logging.properties"
  :noJuliConfig
! set "JAVA_OPTS=%JAVA_OPTS% %LOGGING_CONFIG%"
  
  if not "%LOGGING_MANAGER%" == "" goto noJuliManager
  set LOGGING_MANAGER=-Djava.util.logging.manager=org.apache.juli.ClassLoaderLogManager
  :noJuliManager
! set "JAVA_OPTS=%JAVA_OPTS% %LOGGING_MANAGER%"
  
  rem ----- Execute The Requested Command ---------------------------------------
  
--- 193,204 ----
  if not exist "%CATALINA_BASE%\conf\logging.properties" goto noJuliConfig
  set LOGGING_CONFIG=-Djava.util.logging.config.file="%CATALINA_BASE%\conf\logging.properties"
  :noJuliConfig
! set JAVA_OPTS=%JAVA_OPTS% %LOGGING_CONFIG%
  
  if not "%LOGGING_MANAGER%" == "" goto noJuliManager
  set LOGGING_MANAGER=-Djava.util.logging.manager=org.apache.juli.ClassLoaderLogManager
  :noJuliManager
! set JAVA_OPTS=%JAVA_OPTS% %LOGGING_MANAGER%
  
  rem ----- Execute The Requested Command ---------------------------------------
  
